ui/package.json

```diff
@@ -63,6 +70,7 @@
     "zustand": "^4.1.3"
   },
   "scripts": {
+    "generate": "chmod 777 /root && rm -rf ./src/proto && npx buf generate --output ./src/",
```

Comment:
Why do we need `chmod 777 /root`? The app inside the container is mounted at `/app/src/`.

Reply:
Otherwise, `npx buf generate` will fail because I need to create & read a `/root/.config/buf/config.yaml`. I have no idea how to fix it.

Reply:
I see. I'll take a look later.